Week-7/problem_7.6_.js

```diff
@@ -0,0 +1,37 @@
+class Queue {
+  constructor() {
+    this.s1 = [];
+    this.s2 = [];
+  }
+
+  enQueue(x) {
+    this.s1.push(x);
+  }
+
+  deQueue() {
+    if (this.s1.length == 0 && this.s2.length == 0) {
+      console.log('Q is empty');
+      return;
+    }
+
+    if (this.s2.length == 0) {
+      while (this.s1.length != 0) {
+        this.s2.push(this.s1[0]);
+        this.s1.shift();
+      }
+    }
+
+    let x = this.s2[0];
```

Comment:
what does this do?

Reply:
yeah this is not needed here, I'll remove this.